packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -133,6 +146,32 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     [onFetchAvatarPersonaData]
   );
 
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
+  const uploadedFilesSelector = useSelector(fileUploadsSelector);
+
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
```

Comment:
I'd recommend extracting a new component `FileUploadCards` into its own file.

The benefit is that you need far lesser conditional compilation. You should be able to define that component unconditionally, and then here you need just one conditionally compiled statement:

```ts
/* @conditional-compile-remove-from(stable): FILE_SHARING */
const onRenderAttachedFiels = useMemo(() => <FileUploadCards cancelFileUpload={adapter.cancelFileUpload} />, [adapter.cancelFileUpload]);
```

`FileUploadCards` would call `useSelector(fileUploadsSelector)` internally.

Reply:
The `FileCardGroup` will be rendered by both sendbox as well as received messages. So, we would need to pass files as props for `FileUploadCards` component as received files would be different than uploaded and we would have to add two methods to handle that separately.